pkgs/build-support/go/README.md

```diff
@@ -42,4 +42,7 @@ Based on this, we align on the following policy for toolchain/builder upgrades:
     If the package won't build with that builder anymore, the package is marked broken.
     It is the package maintainers responsibility to fix the package and get it working with a supported Go toolchain.
 
+For the stable release, we recognize that (1) removing a Go version, or updating the `go_latest` or `go` packages to a new Go minor release, would be a breaking change, and (2) some packages will need backports (e.g. for security reasons) that require the latest Go version.
+Therefore, on the stable release, new Go versions will be backported to the `release-2x.xx` branch, but the old versions will remain, and `go`, `buildGoModule`, `go_latest`, and `buildGoLatestModule` will remain unchanged.
```

Comment:
Could you add a sentence that rc versions shouldn't be added to the stable release branch?